ng-package.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
Need to investigate if this file can live at root.

Reply:
I investigated this for the easy change but it throws warnings in the build for the `ng-packagr` Would like to keep it here for now.

Reply:
If / when we have multiple packages we may want to investigate something like this: https://github.com/lerna/lerna